docs/DOCUMENTATION_GUIDE.md

```diff
@@ -0,0 +1,22 @@
+# Documentation Guidelines
+
+## Starting a New Project
+
+The dashboards-anywhere repository hosts multi-projects, this documentation guidelines provides documentaztion guide for the developers who starting a new project. 
+
+From here it's recommended to follow a few patterns.
+
+```
+# Docs Folder Structure
+.
+├── projects
+|      ├── playground
+|      |       ├── design.md
+|      |       └── other doc
+|      └── other porject
+└── DOCUMENTATION_GUIDE.md
+```
+
+## Documentation Format
+
+We suggest to use markdown file as documentation format, it can easily to use or reference by other sites.
```

Comment:
nit: it can be easy to use or reference by other sites.

Reply:
Changed. Thanks